staff/albert-ferrer/precourse/bingo/Bingo.js

```diff
@@ -0,0 +1,132 @@
+function Bingo () {
+
+    var card, user, history;
+    
+    const NUMBERS = 5;
+    const MIN = 0;
+    const MAX = 20;
+    const REPLACE = 'X';
+    
+    function initialize () {   //Function local donde se pide el nombre del usuario. Aparte dos funciones dentro (locales).
+      user = prompt('Hey, what\'s your name?');
+      display(`Let's play, ${user}!`);
+      
+      start();
+    }
+
+    function start () {
+      history = []; 
+      card = generateCard(MIN, MAX, NUMBERS); //  Genera un nuevo cartón
+      
+      display(`The card is: ${card}`);
+      play();
+    }
+  
+    function display (message) {
+      
+      alert('Wellcome to Skylab Bingo deLuxe. ' + message);
+
+    }
+    
+    function play () {        /* Function que comprueba si el numero Random está en la Array. Si esta lo cambiara por 'X' . */
```

Comment:
You use for all comments "/* */". If they are of a single line it is easier to use "//"